src/Dapr.AzureFunctions.Extension/Services/DaprServiceClient.cs

```diff
@@ -174,13 +174,14 @@ static async Task ThrowIfDaprFailure(HttpResponseMessage response)
 
         internal async Task PublishEventAsync(
             string? daprAddress,
-            string? topicName,
+            string name,
+            string topicName,
```

Comment:
this was a bad usage of nullable - we actually rely on these being non-null here.